quint/io-cli-tests.md

```diff
@@ -1481,3 +1476,25 @@ won(X) was witnessed in 99 trace(s) out of 100 explored (99.00%)
 stalemate was witnessed in 1 trace(s) out of 100 explored (1.00%)
 Use --seed=0x2b442ab439177 to reproduce.
 ```
+
+### NEW TEST: Run outputs ITF without verbosity flag (should show new default behavior)
+
+<!-- !test in run itf no verbosity -->
+```
+quint run --out-itf=out-itf-novb.itf.json --max-steps=5 --seed=123 \
+  --invariant=totalSupplyDoesNotOverflowInv \
+  --verbosity=0 \
+  ../examples/tutorials/coin.qnt
+cat out-itf-novb.itf.json | jq '.states[0]."balances"."#map"[0]'
+rm out-itf-novb.itf.json
```

Comment:
Here, we should call `run` without `--verbosity` and capture the output to check it
```suggestion
output=$(
quint run --out-itf=out-itf.itf.json --max-steps=5 --seed=123 \
  --invariant=totalSupplyDoesNotOverflowInv \
  ../examples/tutorials/coin.qnt 2>&1)
exit_code=$?
echo "$output" | sed -e 's/([0-9]*ms)/(duration)/g' -e 's#^.*coin.qnt#      HOME/coin.qnt#g'
exit $exit_code
```